src/Product/Receipt/ReceiptV5Document.php

```diff
@@ -0,0 +1,186 @@
+            $pageId
+        );
+        $this->date = new DateField(
+            $rawPrediction["date"],
+            $pageId
+        );
+        $this->documentType = new ClassificationField(
+            $rawPrediction["document_type"],
+            $pageId
+        );
+        $this->lineItems = new ReceiptV5LineItems(
+            $rawPrediction["line_items"],
+            $pageId
+        );
+        $this->locale = new LocaleField(
+            $rawPrediction["locale"],
+            $pageId
+        );
+        $this->subcategory = new ClassificationField(
+            $rawPrediction["subcategory"],
+            $pageId
+        );
+        $this->supplierAddress = new StringField(
+            $rawPrediction["supplier_address"],
+            $pageId
+        );
+        $this->supplierCompanyRegistrations = $rawPrediction["supplier_company_registrations"] == null ? [] : array_map(
+            fn ($prediction) => new CompanyRegistrationField($prediction, $pageId),
+            $rawPrediction["supplier_company_registrations"]
+        );
+        $this->supplierName = new StringField(
+            $rawPrediction["supplier_name"],
+            $pageId
+        );
+        $this->supplierPhoneNumber = new StringField(
+            $rawPrediction["supplier_phone_number"],
+            $pageId
+        );
+        $this->taxes = new Taxes(
+            $rawPrediction["taxes"],
+            $pageId
+        );
+        $this->time = new StringField(
+            $rawPrediction["time"],
+            $pageId
+        );
+        $this->tip = new AmountField(
+            $rawPrediction["tip"],
+            $pageId
+        );
+        $this->totalAmount = new AmountField(
+            $rawPrediction["total_amount"],
+            $pageId
+        );
+        $this->totalNet = new AmountField(
+            $rawPrediction["total_net"],
+            $pageId
+        );
+        $this->totalTax = new AmountField(
+            $rawPrediction["total_tax"],
+            $pageId
+        );
+    }
+
+    /**
+     * @return string String representation.
+     */
+    public function __toString(): string
+    {
+        $supplierCompanyRegistrations = implode(
+            "\n                                 ",
+            $this->supplierCompanyRegistrations
+        );
+        $lineItemsSummary = $this->lineItems->lineItemsToStr();
+
+        $outStr = ":Expense Locale: $this->locale
+:Purchase Category: $this->category
+:Purchase Subcategory: $this->subcategory
+:Document Type: $this->documentType
+:Purchase Date: $this->date
+:Purchase Time: $this->time
+:Total Amount: $this->totalAmount
+:Total Net: $this->totalNet
+:Total Tax: $this->totalTax
+:Tip and Gratuity: $this->tip
+:Taxes: $this->taxes
+:Supplier Name: $this->supplierName
+:Supplier Company Registrations: $supplierCompanyRegistrations
+:Supplier Address: $this->supplierAddress
+:Supplier Phone Number: $this->supplierPhoneNumber
+:Line Items: $lineItemsSummary
```

Comment:
lineItems should be stringable

```suggestion
:Line Items: $this->lineItems
```